File: app/src/main/java/cc/smartcash/wallet/Services/SmartTextService.java
package cc.smartcash.wallet.Services;

import cc.smartcash.wallet.Models.SmartTextRequest;
import cc.smartcash.wallet.Models.SmartTextRoot;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Header;
import retrofit2.http.POST;

public interface SmartTextService {

    @POST("apiorder/createorder")
    Call<SmartTextRoot> sentPayment(@Header("Authorization-Token") String auth, @Body SmartTextRequest sendPayment);

}

File: app/src/main/java/cc/smartcash/wallet/Services/SmartTextService.kt
package cc.smartcash.wallet.Services

import cc.smartcash.wallet.Models.SmartTextRequest
import cc.smartcash.wallet.Models.SmartTextRoot
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Header
import retrofit2.http.POST

interface SmartTextService {

    @POST("apiorder/createorder")
    fun sentPayment(@Header("Authorization-Token") auth: String, @Body sendPayment: SmartTextRequest): Call<SmartTextRoot>

}